src/main/java/de/hpi/bpt/argos/util/RestEndpointUtilImpl.java

```diff
@@ -112,20 +112,25 @@ public String executeRequest(Logger logger, Request request, Response response,
 		try {
 			response.body((String) route.handle(request, response));
 		} catch (HaltException e) {
-			logSendingResponse(logger, request, e.statusCode(), e.body());
-			throw e;
```

Comment:
Actually no. This way I can handle all responses myself and dont have to hand them back to the framework. (This becomes important, when setting the response type in error-cases)